micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/QuickStart/QuickSetup.js

```
@@ -0,0 +1,143 @@
+        <div style={{ marginTop: "1rem" }}>
+        {actions && (
+          <ListTag>
+          {actions.map((action, index) => (
+            <li key={index} style={{ listStyleType: ListTag === "ul" ? "disc" : "auto", margin: "8px 0" }}>
+              {isLabelLink ? (
+                action?.label ? (
+                  <a
+                  className="quickLink"
+                  href="#"
+                  onClick={(e) => handleClick(e, `${window.location.host}/${window?.globalPath}/${tenantId}/${action?.link}`)}
+                >
```

Biome

[error] Use a button element instead of an a element.

Anchor elements should only be used for default sections or page navigation
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)